src/sentry/models/release.py

```diff
@@ -839,14 +888,16 @@ def add_project(self, project):
 
         try:
             with atomic_transaction(using=router.db_for_write(ReleaseProject)):
-                created = ReleaseProject.objects.get_or_create(project=project, release=self)[1]
+                obj, created = ReleaseProject.objects.get_or_create(project=project, release=self)[
```

Comment:
This is an unrelated change 🤔 

I'll move this to a separate PR.

get_or_create actually returns a tuple and we've previously been utilizing this incorrectly